lib/commons/dom/visibility-functions.js

```diff
@@ -0,0 +1,171 @@
+import {
+  getScroll,
+  closest,
+  getRootNode,
+  querySelectorAll,
+  escapeSelector
+} from '../../core/utils';
+
+const clipRegex = /rect\s*\(([0-9]+)px,?\s*([0-9]+)px,?\s*([0-9]+)px,?\s*([0-9]+)px\s*\)/;
+const clipPathRegex = /(\w+)\((\d+)/;
+
+/**
+ * Determine if an element is natively hidden
+ * @param {VirtualNode} vNode
+ * @return {Boolean}
+ */
+export function nativelyHidden(vNode) {
```

Comment:
I wonder if we should expand this list. There are more elements that are hidden by default. `param`, `option`, `title`, etc. We could consider putting a property on the standard for this instead. We have other rules that use this kind of list too I believe.

Reply:
I like the idea of adding it to the html element standard

Reply:
Created https://github.com/dequelabs/axe-core/issues/3641